components/app-core/frontend/i18n/en/endpoints.json

```diff
@@ -1,13 +1,6 @@
 {
   "endpoints": {
     "unknown": "Unknown",
-    "register": "Register",

```

Comment:
These were a generic set that aren’t referenced anywhere. The registration info is now per plugin, see cf.registration.<x> strings.